it/src/test/scala/com/wavesplatform/it/sync/DataTransactionSuite.scala

```diff
@@ -90,46 +88,50 @@ class DataTransactionSuite extends BaseTransactionSuite {
   test("data definition and retrieval") {
     // define first data item
     val item1 = IntegerDataEntry("int", 8)
-    val tx1 = sender.putData(secondAddress, List(item1), fee).id
+    val listItem1 = List(item1)
+    val tx1 = sender.putData(secondAddress, listItem1, calcDataFee(listItem1)).id
     nodes.waitForHeightAraiseAndTxPresent(tx1)
 
     sender.getData(secondAddress, "int") shouldBe item1
-    sender.getData(secondAddress) shouldBe List(item1)
+    sender.getData(secondAddress) shouldBe listItem1
 
     // define another one
     val item2 = BooleanDataEntry("bool", true)
-    val tx2 = sender.putData(secondAddress, List(item2), fee).id
+    val listItem2 = List(item2)
+    val tx2 = sender.putData(secondAddress, listItem2, calcDataFee(listItem2)).id
     nodes.waitForHeightAraiseAndTxPresent(tx2)
 
     sender.getData(secondAddress, "int") shouldBe item1
     sender.getData(secondAddress, "bool") shouldBe item2
-    sender.getData(secondAddress) shouldBe List(item2, item1)
+    sender.getData(secondAddress) shouldBe List.concat(listItem2, listItem1)
```

Comment:
not nessessary, but `listItem2 ++ listItem1` will do the same. Also mind the naming, `dataList1` and `dataList2` are more descriptive.